### DISCUSSION TITLE
Need feedback on making navbar background transparent/disappear when collapsed.

### DISCUSSION THREAD

**Discussion by SamuelMiller** [twbs/bootstrap#36539] [Q&A]
In my Codepen example of what I am trying to achieve, the navbar fully disappears "into" the toggler icon when the navbar reaches its breakpoint and collapses--the navbar background becomes transparent.  When the icon is toggled the background reappears for the collapse menu.
I have created the basic effect by using a custom css media query :
```
@media (max-width: 576px) {
	.navbar.bg-light {
		--bs-bg-opacity: 0;
	}

	.navbar-collapse {
		position: absolute;
		left: 0;
		top: 0;
		width: 100%;
		z-index: -10;
		--bs-gutter-x: 1.5rem;
		padding-right: calc(var(--bs-gutter-x) * .5);
		padding-left: calc(var(--bs-gutter-x) * .5);
	}
}
```
See working example:
https://codepen.io/sam-miller-the-flexboxer/pen/poaQvKm

I am curious to hear from others if there is a better approach or ways to improve my Codepen example? How about a way to make the text flow around the fixed toggle icon when scrolling?

**Comment by ghost** [comment]
you can enclose it in a fluid container, so it doesn't disappear completely
